src/domains/chart/selectors.ts

```diff
@@ -61,3 +61,19 @@ export const selectNameOfAnyFetchingChart = createSelector(
   (chartsState) => Object.values(chartsState)
     .find((chartState) => chartState.isFetchingData)?.chartDetails?.id,
 )
+
+export const selectAmountOfSnapshotsFetched = createSelector(
+  selectChartsState,
+  (chartsState) => Object.values(chartsState)
+  // count the nr of "success" or "failure" charts
+    .map((chartState) => Boolean(chartState.snapshotData))
+    .reduce((acc, value) => acc + Number(value), 0),
```

Comment:
I'm not sure I understand. So, the `value` is a boolean type (from prev `map`), and we cast it to Number?

Reply:
yes, i need the number of `chartStates` where the boolean is true, do you think it could be done differently, so it's more clear?
> // count the nr of "success" or "failure" charts

But this ^^ comment is wrong, it's a result of copy/paste, that's probably what is confusing

Reply:
so why not to do than in one `reduce` iteration?

Reply:
`.reduce((acc, chartState) => acc + !!chartState ? 1 : 0 , 0),`

Reply:
yes, that looks better! fixed